packages/php-table-backend-utils/src/Schema/Snowflake/SnowflakeSchemaReflection.php

```diff
@@ -52,4 +56,99 @@ public function getViewsNames(): array
 
         return array_map(static fn($table) => $table['name'], $tables);
     }
+
+    /**
+     * @return array<string, SnowflakeTableDefinition>
+     */
+    public function getDefinitions(): array
+    {
+        $informationsQuery = sprintf(
+            'SELECT TABLE_NAME, TABLE_TYPE, BYTES, ROW_COUNT '.
+            'FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_SCHEMA = \'%s\' ORDER BY TABLE_NAME;',
```

Comment:
tohle by se melo escapovat jako `SnowflakeQuote::quoteSingleIdentifier($this->schemaName)`, nemusel bys pak mit `\'%s\'` 

udelej to pls jeste takto, at to tady je jednotne

Reply:
pravda, upraveno